bsuite/experiments/mdp_playground/analysis.py

```diff
@@ -0,0 +1,75 @@
+# python3
+# pylint: disable=g-bad-file-header
+# Copyright 2020 #TODO ... All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or  implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Analysis for MDP Playground."""
+
+###TODO change to mdpp stuff below
+from typing import Sequence
+
+from bsuite.experiments.mdp_playground import sweep
+from bsuite.utils import plotting
+import numpy as np
+import pandas as pd
+import plotnine as gg
+
+NUM_EPISODES = sweep.NUM_EPISODES
+BASE_REGRET = 100
+GOOD_EPISODE = 50
+TAGS = sweep.TAGS
+
+def score(df: pd.DataFrame) -> float:
+  """Output a score for MDP Playground."""
+  df = mdpp_preprocess(df_in=df)
+  regret_score = plotting.ave_regret_score(
+      df, baseline_regret=BASE_REGRET, episode=NUM_EPISODES)
+
+  norm_score = 1.0 * regret_score # 2.5 was heuristically chosen value to get Sonnet DQN to score approx. 0.75, so that better algorithms like Rainbow can get score close to 1. With a bigger NN this would mean an unclipped score of 1.1 for Sonnet DQN, which is fair I think. However, a2c_rnn even reached 2.0 on this scale. DQN may be not performing as well because its epsilon is not annealed to 0.
```

Comment:
I don't understand the comment I'm afraid. Was the value here 2.5 at some other point>